masterlist.yaml

```diff
@@ -1570,6 +1570,10 @@ plugins:
         util: 'SSEEdit v4.0.3'
 
 ###### Fixes ######
+  - name: 'BugfixCompilation.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/33449/' ]
+    after: [ 'Relationship Dialogue Overhaul.esp' ]
```

Comment:
From [Bugfix Compilation](https://www.nexusmods.com/skyrimspecialedition/mods/33449) description page.

![image](https://user-images.githubusercontent.com/1944639/77527194-a23e8980-6e83-11ea-8c37-c25060982998.png)

> Compatibility
> 
> This should be compatible with just about everything. It is a low priority mod, meaning that it should be at the very top of your load order. Let other mods overwrite it.

![image](https://user-images.githubusercontent.com/1944639/77527380-e92c7f00-6e83-11ea-91eb-60bd6633363b.png)

> Hunters Not Bandits by raccoondance
> 
> The regular version is included. In case you're using RDO, just let it overwrite this compilation or download "Hunters Not Bandits for RDO" and place it after both mods.

Based on that and the overall content of the mod.
I would place it early as suggested in the `Fixes & Resources` group.

```suggestion
    group: *fixesGroup
```